recipes/qcbor/all/conanfile.py

```diff
@@ -54,6 +52,11 @@ def source(self):
 
     def generate(self):
         tc = CMakeToolchain(self)
+        if Version(self.version) >= "1.2":
+            tc.variables["QCBOR_OPT_DISABLE_FLOAT_HW_USE"] = self.options.disable_float in ["HW_USE", "PREFERRED", "ALL"]
+            tc.variables["QCBOR_OPT_DISABLE_FLOAT_PREFERRED"] = self.options.disable_float in ["PREFERRED", "ALL"]
+            tc.variables["QCBOR_OPT_DISABLE_FLOAT_ALL"] = self.options.disable_float == "ALL"
+            tc.cache_variables["CMAKE_POLICY_DEFAULT_CMP0077"] = "NEW"
```

Comment:
this should be fixed in 1.54

Reply:
Right, it has been fixed by https://github.com/conan-io/conan/pull/12401

Reply:
@toge currently we are running Conan 1.57.0 here in CCI: https://github.com/conan-io/conan-center-index/blob/master/.c3i/config_v1.yml#L6 so you could update `required_conan_version` to 1.54.0 and remove `CMAKE_POLICY_DEFAULT_CMP0077` line